webviz_subsurface/_abbreviations/reservoir_simulation.py

```diff
@@ -0,0 +1,71 @@
+import json
+import pathlib
+import warnings
+
+
+_DATA_PATH = pathlib.Path(__file__).parent.absolute() / "abbreviation_data"
+
+SIMULATION_VECTOR_TERMINOLOGY = json.loads(
+    (_DATA_PATH / "reservoir_simulation_vectors.json").read_text()
+)
+
+RESERVOIR_SIMULATION_UNIT_TERMINOLOGY = json.loads(
+    (_DATA_PATH / "reservoir_simulation_unit_terminology.json").read_text()
+)
+
+
+def simulation_unit_reformat(ecl_unit: str, unit_set: str = "METRIC"):
+    """Returns the simulation unit in a different, more human friendly, format if possible,
+    otherwise returns the simulation unit.
+    * `ecl_unit`: Reservoir simulation vector unit to reformat
+    * `unit_set`: Currently only valid option is the default "METRIC" (defined as in Eclipse E100)
+    """
+    try:
+        unit = RESERVOIR_SIMULATION_UNIT_TERMINOLOGY[unit_set][ecl_unit]
+    except KeyError:
+        unit = ""
+
+    return unit
```

Comment:
If we are okey with `KeyError` on unknown `unit_set`, this can be simplified to
```python
return RESERVOIR_SIMULATION_UNIT_TERMINOLOGY[unit_set].get(ecl_unit, ecl_unit)
```
In any case, shouldn't `ecl_unit` be returned and not an empty string? (If not more human friendly unit is defined).

Reply:
I agree